lochness/box/__init__.py

```diff
@@ -65,16 +65,38 @@ def get_box_object_based_on_name(client:boxsdk.client,
 
     Returns:
         box_folder_object
+
     '''
     box_folder_name = str(box_folder_name)
 
+    # if box root is given as path - eg) box_folder_name == 'example/PronetLA'
+    if '/' in box_folder_name:
+        # remove leading '/' from the path string
+        box_folder_name = box_folder_name[1:] if box_folder_name[0] == '/' \
```

Comment:
Is the `box_folder_name` a list here? If not `[1:]` and `[0]` indices are invalid. I see it as a string only.

Reply:
no `box_folder_name` is `str` as noted in the function annotation.

Reply:
> If not [1:] and [0] indices are invalid.

Reply:
I'm doing something similar to the example below.

```
In [1]: a = 'abcdefgh'

In [2]: a = a[1:] if a[0] == '/' else a

In [3]: a
Out[3]: 'abcdefgh'

In [4]: a = '/abcdefgh'

In [5]: a = a[1:] if a[0] == '/' else a

In [6]: a
Out[6]: 'abcdefgh'
```

Reply:
Okay, answer this--in the code comment you have noted `'example/PronetLA'`. How does your logic help to get just `PronetLA` from that?

Reply:
You want the last string only--you just do `os.path.split(box_folder_name)[-1]`--valid even when your `box_folder_name` does not have a `/`.

If you do as I said, newly added lines 72-92 are useless.

Reply:
The lines here are added to take care of any user providing the box source root path with `/` in the beginning of the source root path given in the `config.yml`, as a proficient linux user may feel comfortable to do so.

Correct, if the lochness is used exactly how we provide in the template we provide, these lines will not be used. I'm trying to provide a flexibility here.

Reply:
>to take care of any user providing the box source root path with / in the beginning of the source root path given

Okay I understand it will take care of `box_folder_name=/PronetLA` and make it `PronetLA`. Yet, [this](https://github.com/PREDICT-DPACC/lochness/pull/31/files#r658871042) comment needs an answer.

Reply:
Thank you, sorry about not understanding earlier. One last suggestion, would you like to note ` '/example/PronetLA' at:

```# if box root is given as path - eg) box_folder_name == 'example/PronetLA'```